graphql.links.menu.yml

```diff
@@ -3,3 +3,9 @@ graphql.config:
   description: 'Configuration options for the GraphQL module.'
   route_name: entity.graphql_server.collection
   parent: system.admin_config_services
+
+graphql.query_maps:
```

Comment:
All features (including query maps) should be scoped per Server/Schema. We will need the ability to configure per-schema "features" for other things in the future too so we should come with a pluggable pattern for this.

E.g. one thing we would also build with a pluggable approach would be logging and performance tracing for individual schemas.